src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -242,6 +241,16 @@ private native void enableThreadNetwork(
 
   public static native void setKeyValueStoreManager(KeyValueStoreManager manager);
 
+  public static native void setServiceResolver(ServiceResolver resolver);
+
+  public static native void handleServiceResolve(
+      String instanceName,
+      String serviceType,
+      String address,
+      int port,
+      long callback,
```

Comment:
Here and in the JNI code, can we rename `callback` -> `callbackHandle` to make it clear?